test/e2e/storage/regional_pd.go

```diff
@@ -44,6 +48,7 @@ import (
 const (
 	pvDeletionTimeout       = 3 * time.Minute
 	statefulSetReadyTimeout = 3 * time.Minute
+	taintKey                = "zoneTaint"
```

Comment:
I would make this key name more specific to make it easier to debug. Maybe use the test namespace name?